internal/worker/memory_test.go

```
@@ -0,0 +1,879 @@
+		if got := sessionsTokenFromSecret(tc.secret); got != tc.want {
+			t.Errorf("%s: %q, want %q", name, got, tc.want)
+		}
+	}
+}
+
+func equalStrings(a, b []string) bool {
+	if len(a) != len(b) {
+		return false
+	}
+	for i := range a {
+		if a[i] != b[i] {
```

golangci-lint (2.12.2)

[low] G602: slice index out of range

(gosec)